op-e2e/setup.go

```diff
@@ -815,7 +811,7 @@ func (cfg SystemConfig) Start(t *testing.T, _opts ...SystemConfigOption) (*Syste
 	// batcher defaults if unset
 	batcherMaxL1TxSizeBytes := cfg.BatcherMaxL1TxSizeBytes
 	if batcherMaxL1TxSizeBytes == 0 {
-		batcherMaxL1TxSizeBytes = 240_000
+		batcherMaxL1TxSizeBytes = 120_000
```

Comment:
`240_000` just never made sense and I just switched it to the default that we also use in production. Since all e2e test never create such large channels/frames and the batcher is configured to instantly send each block as batch, no test ever gets close to this limit anyways.